test/integration/transaction.test.js

```diff
@@ -527,6 +527,93 @@ if (current.dialect.supports.transactions) {
 
     }
 
+    describe('isolation levels', () => {
+      it('should read the most recent committed rows when using the READ COMMITTED isolation level', function() {
+        const User = this.sequelize.define('user', {
+          username: Support.Sequelize.STRING
+        });
+
+        return expect(
+          this.sequelize.sync({ force: true }).then(() => {
+            return this.sequelize.transaction({ isolationLevel: Transaction.ISOLATION_LEVELS.READ_COMMITTED }, transaction => {
+              return User.findAll({ transaction }).then(users => {
+                expect( users ).to.have.lengthOf(0);
+              }).then(() => {
+                // Create a User outside of the transaction
+                return User.create({ username: 'jan' }).then(() => { 
+                  // We should see the created user inside the transaction
+                  return User.findAll({ transaction }).then(users => {
+                    expect( users ).to.have.lengthOf(1);
+                  });
+                });
+              });
+            });
+          })
+        ).to.eventually.be.fulfilled;
+      });
+
+      // mssql is excluded because it implements REPREATABLE READ using locks rather than a snapshot, and will see the new row
+      if (['mariadb', 'mysql', 'postgres', 'postgres-native'].includes(dialect)) {
```

Comment:
Use blacklist like `!['mssql'].includes(dialect)`, this will allow future dialects to accommodate for this test